### PR TITLE
update to Drupal 10

### PR DESCRIPTION
**Pull request by Matvey-Davidson**
https://lilt.atlassian.net/browse/ENG-6479

I updated the configuration files to support Drupal 10
It wasn't any problematic modules

### CODE REVIEW

File: tmgmt_lilt.info.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Lilt Translator
 description: 'The Lilt translator module that integrates with the Translation Management Tool.'
-core_version_requirement: ^8.8 || ^9
+core_version_requirement: ^9  || ^10
```
Comment (tgallant) [inline tmgmt_lilt.info.yml]: should we still support 8.8 here?

```suggestion
core_version_requirement: ^8.8 || ^9  || ^10
```

File: composer.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
     }
   ],
   "require": {
+    "drupal/core": "^9 || ^10",
```
Comment (tgallant) [inline composer.json]: same here, should we include `^8.8`?

**Review by tgallant — COMMENTED** [review]
have you been able to try running this locally? we have a separate repo here for creating a local drupal environment. maybe there might be something to update there to support using drupal v10

**Comment by Matvey-Davidson** [comment]
> have you been able to try running this locally? we have a separate repo here for creating a local drupal environment. maybe there might be something to update there to support using drupal v10

Yes, I did it on https://github.com/lilt/lilt_drupal_env , changed the branch to mine in require and executed the unit tests
For `^8` addition it worked too

File: tmgmt_lilt.info.yml (hunk 1 above)
Comment (Matvey-Davidson) [reply tmgmt_lilt.info.yml]: Sure, I will update and test it

File: composer.json (hunk 2 above)
Comment (Matvey-Davidson) [reply composer.json]: Sure, I will update and test it

**Review by tgallant — APPROVED** [review]

**Pull request merged**